data/org.ArcticaProject.arctica-greeter.gschema.xml

```diff
@@ -335,5 +331,13 @@
       <default>1</default>
       <summary>The vertical offset of the logo from the edge of the screen in grid units.</summary>
     </key>
+    <key name="error-below-entry" type="b">
+      <default>false</default>
+      <summary>Whether to display the error label below the login entry.</summary>
+    </key>
+    <key name="menubar-bgcolor" type="s">
+      <default>'rgba(0,0,0,0.2)'</default>
```

Comment:
This introduces a new color naming scheme for menubar compared to the various boxes we have in the greeter UI. Can't we keep the menubar alpha and have an RBG ``#123456`` value here?

Reply:
Done.